lib/src/hierarchy_view.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dump_viz.hierarchy_view;
```

Comment:
typically I would include `src` in the library name as well to ensure uniqueness